doc/rst/developer/bestPractices/EditionsDev.rst

```diff
@@ -18,8 +18,111 @@ When creating new editions, be sure to:
+
+    A stable feature should only have its behavior change in the ``preview``
+    edition. The ``preview`` edition collects changes that will be included in
+    the next edition.
+
+    .. note::
+
+        An unimplemented feature is the
+        `--flag-edition-changes <TransitionFlag>`_ flag, which will warn users
+        in a given edition about code that will need to be migrated in the next
+        edition.
+
+* **Deprecation (and subsequent removal) of a stable feature** (`Breaking Change`)
+
+    A stable feature should only be deprecated in the ``preview`` edition. The
+    ``preview`` edition collects changes that will be included in the next
+    edition.
+
+    .. note::
+
+        An unimplemented feature is the
+        `--flag-edition-changes <TransitionFlag>`_ flag, which will warn users
+        in a given edition about code that will need to be migrated in the next
+        edition.
+
+* **Changing the behavior of an unstable feature** (`Non-Breaking Change`)
+
+    Unstable features are not considered part of an edition, and so their
+    behavior can be changed at any time without being associated with an
+    edition. However, the first Chapel compiler release to change the behavior
+    of an unstable feature should warn users that the feature has changed in
+    some way. Later Chapel compiler releases may remove the warning.
+
+* **Deprecation (and subsequent removal) of an unstable feature** (`Non-Breaking Change`)
+
+    Unstable features are not considered part of an edition, and so they can be
+    deprecated at any time without being associated with an edition. However,
+    the first Chapel compiler release to deprecate or remove an unstable
+    feature should warn users that the feature has been deprecated or removed.
+    Later Chapel compiler releases may remove the warning and the feature.
+    Unstable features should not be removed without warning.
+
+* **Addition of a new feature** (`Non-Breaking Change`)
+
+    New features may be added at any time without being associated with an
+    edition, as long as they are added as unstable features. The new feature
+    can be added as an unstable feature to any Chapel compiler release. This
+    makes the feature available to users who wish to use it, without affecting
+    the edition. When we feel the new feature is ready to be stabilized, it can
+    be marked as stable in a new edition (either being added to the ``preview``
+    edition staging ground or directly to the next edition).
+
+    Users who make use of the new feature should get an unstable warning,
+    unless they are using an edition of the language that includes the feature
+    as stable.
+
+In summary, if a user is using only stable features of a given language
+edition, two different versions of the compiler that support that edition
+should behave the same way (baring bug fixes, performance improvements, etc).
+Non-breaking changes to stable features should still be associated with a
```

Comment:
Huh? Changes to stable features are breaking? I suspect this sentence doesn't say what you want to. Otherwise it would say, changing a comment on a stable feature should be associated with an edition. 

Maybe you meant "Breaking changes..." ?